_specs/ECIP-TBD.md

```diff
@@ -0,0 +1,36 @@
+---
+ecip: ECIP number TBD
+title: Proposal for owner, member, editor, & author removal
+author: Donald McIntyre (@TokenHash)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/273
+status: Draft
+type: Meta
+created: 2020-01-20
+license: Apache-1
+---
+
+### Proposal
+
+I propose Wei Tang (@sorpaas) be expelled from the Ethereum Classic organization as a 'member' or 'owner' whatever is his role there. I also propose for him to be taken out as an 'editor' in the ECIP process, namely ECIP-1000, and that ECIP-1000 be replaced with a new clean 'process' ECIP with a new author, so Sorpaas is removed from that position as well.
+
+### Rationale
```

Comment:
This section requires examples / links to screenshots otherwise it's impossible to review these claims against @sorpaas.

Reply:
Doxxing:

![Screen Shot 2020-01-19 at 11 01 05 AM](https://user-images.githubusercontent.com/29529586/72753069-1fe2d200-3b81-11ea-8a77-b69f1296639d.png)

Reply:
Example of lying claiming that an 'accepted' ECIP cannot be changed:

![A](https://user-images.githubusercontent.com/29529586/72753234-8ff15800-3b81-11ea-9d81-519f8be592a3.png)

Reply:
Example of accusing other contributors of "bypassing" an imaginary rule in the process, and ECIP process as a whole, contributors just propose ideas:

![B](https://user-images.githubusercontent.com/29529586/72753313-de9ef200-3b81-11ea-9830-037735e28de0.png)

Reply:
Example of intimidation of other contributors accusing them of proposing things that will provoke vulnerabilities:

![C](https://user-images.githubusercontent.com/29529586/72753604-946a4080-3b82-11ea-95af-f37eaadf1a4f.png)

Reply:
Example where he describes the immutability of the blockchain is analogous to the ECIP process:

![D](https://user-images.githubusercontent.com/29529586/72753909-76e9a680-3b83-11ea-851d-f2c5b2a93786.png)

Reply:
Example where he makes a false analogy and accuses another contributor that he would not be on the side of immutability and would have been in favor of TheDAO hard fork, making, again, a false equivalence that the ECIP process is identical as the operating network:

![E](https://user-images.githubusercontent.com/29529586/72754011-e069b500-3b83-11ea-83fd-8949f9449320.png)

Reply:
Example of making the false equivalence between the ECIP process and the operating blockchain, and intimidating other contributors by re-phrasing their comments, falsely claiming they mean something they don't and subsequently asking the contributor to shut up if he doesn't want his comments to be re-interpreted with false assumptions:

![F](https://user-images.githubusercontent.com/29529586/72754188-5a01a300-3b84-11ea-9576-d6fcd44ececc.png)

Reply:
Lying about rules and text on ECIP-1000 that don't exist, trying to stop changes to his or other ECIPs:

![G](https://user-images.githubusercontent.com/29529586/72754356-c8466580-3b84-11ea-9b02-609f1b12414b.png)

Reply:
Pointing again that a rule or text exists, but was never able to demonstrate because they don't actually exist. Subsequently, insulting the other contributor again:

![H](https://user-images.githubusercontent.com/29529586/72754495-3559fb00-3b85-11ea-9b44-9c59008dfb59.png)

Reply:
Repeatedly falsely claiming other contributors don't want to follow or want to bypass the ECIP process as way to intimidate, change the conversation, and stop other changes making up reasons. 

Claiming that an "irregular" path is created, making the false analogy to theDAO hard fork "irregular state" transition:

![I](https://user-images.githubusercontent.com/29529586/72754762-fb3d2900-3b85-11ea-92ba-297956990eb6.png)

Reply:
...many more examples thru the last few years...the above was only one conversation, but he has been harassing and abusing his quadruple position in ETC in many other ways...